db/migrate/20150622194754_create_group_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGroupUsers < ActiveRecord::Migration
+  def change
+    create_table :group_users do |t|
+      t.integer :user_id
+      t.integer :group_id
+      t.boolean :admin
```

Comment:
Lägga in en permission för unika grupper?
Det är t.ex. en faddergrupp där alla faddrar ska få tillåtelse att lägga till andra.